build-docker.sh

```
@@ -484,13 +490,14 @@ echo "Fingerprints:"
 # Display core and legacy fingerprints (if built)
 for VARIANT in core legacy; do
   for MODEL in ${MODELS[@]}; do
-    for DIRSUFFIX in "" "-bitcoinonly"; do
+    for DIRSUFFIX in "" "-bitcoinonly" $DIRSUFFIX_OVERRIDE; do
       BUILD_DIR=build/${VARIANT}-${MODEL}${DIRSUFFIX}
       for file in $BUILD_DIR/*/*.fingerprint; do
```

Shellcheck (0.11.0)

[info] Quote expansions in this for loop glob to prevent wordsplitting, e.g. "$dir"/*.txt .

(SC2231)